### PR TITLE
Changed the project structure and added avatar style input

### PR DESCRIPTION
**Pull request by imashishks**
The branch contains the following updates:

1) The initial project structure update and addition of avatar component
2) Adding avatar style input attribute to the avatar component
3) Adding style input attribute to the avatar component

### CODE REVIEW

File: projects/avataaars-angular/src/lib/components/avatar/avatar.component.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  @Input() style: object ={};
+  @Input() avatarStyle = AvatarStyle.Circle;
+  
+  constructor() {
+   
+  }
+    
+  ngOnInit(): void {
+     console.log(this.avatarStyle);
+     console.log(this.style);
```
Comment (atish31) [inline projects/avataaars-angular/src/lib/components/avatar/avatar.component.ts]: Let's get rid of console statements here.

File: projects/avataaars-angular/src/public-api.ts [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
 
+
+
 export * from './lib/avataaars-angular.service';
-export * from './lib/avataaars-angular.component';
+export * from './lib/components/avatar/avatar.component';
 export * from './lib/avataaars-angular.module';
+
+
+// enums
```
Comment (atish31) [inline projects/avataaars-angular/src/public-api.ts]: lets get rid of this commented line and reduce some white space lines here

File: projects/avataaars-angular/src/lib/components/avatar/avatar.component.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  templateUrl:'./avatar.component.html',
+  styles: [
+  ]
+})
+export class AvatarComponent implements OnInit {
+  @Input() style: object ={};
+  @Input() avatarStyle = AvatarStyle.Circle;
+  
+  constructor() {
+
```
Comment (atish31) [inline projects/avataaars-angular/src/lib/components/avatar/avatar.component.ts]: lets get rid of unnecessary white space lines.

**Review by atish31 — CHANGES_REQUESTED** [review]

File: projects/avataaars-angular/src/lib/components/avatar/avatar.component.ts (hunk 3 above)
Comment (imashishks) [reply projects/avataaars-angular/src/lib/components/avatar/avatar.component.ts]: Done

File: projects/avataaars-angular/src/public-api.ts (hunk 2 above)
Comment (imashishks) [reply projects/avataaars-angular/src/public-api.ts]: Done

**Review by atish31 — APPROVED** [review]

**Pull request merged**